### PR TITLE
♿ [#2344] Add text alternatives for meaningful status-icons

### PR DESCRIPTION
**Pull request by jiromaykin**
icons need to remain hidden for screenreaders (aria-hidden=true) because material-icons and font-awesome etc. do not provide proper meaningful text, so to add meaning we add a span with the visually hidden sr-only class.

Taiga: https://taiga.maykinmedia.nl/project/open-inwoner/task/2344

### CODE REVIEW

**Comment by jiromaykin** [comment]
@alextreme suggesties voor betere teksten zijn welkom.

File: src/open_inwoner/templates/pages/cases/statuses.html [hunk 1]
```diff
@@ -13,6 +13,7 @@ <h2 class="utrecht-heading-2" id="statuses">{% trans 'Status' %}</h2>
                 {% firstof case.id|slugify as id %}
                 {% firstof 'content-'|add:id|add:'-id' as content_id %}
                     {% icon icon="circle" extra_classes="status-step" %}
+                    <span class="sr-only">{% trans 'Niet voltooid' %}</span>
```
[the remark this answers is not in the document]
Comment (pi-sigma) [reply src/open_inwoner/templates/pages/cases/statuses.html:16]: Let's stick to one kind of quotes unless we need to depart.

File: src/open_inwoner/templates/pages/cases/statuses.html [hunk 2]
```diff
@@ -47,6 +48,7 @@ <h3 class="utrecht-heading-3 utrecht-accordion__header">
                 {% firstof case.id|slugify as id %}
                 {% firstof 'completed-'|add:id|add:'-id' as completed_id %}
                     {% icon icon="task_alt" %}
+                    <span class="sr-only">{% trans 'Voltooid' %}</span>
```
Comment (pi-sigma) [inline src/open_inwoner/templates/pages/cases/statuses.html:51]:
```suggestion
                    <span class="sr-only">{% trans "Voltooid" %}</span>
```

File: src/open_inwoner/templates/pages/cases/statuses.html [hunk 3]
```diff
@@ -67,6 +69,7 @@ <h3 class="utrecht-heading-3 utrecht-accordion__header">
         {% if case.second_status_preview %}
             <li class="status--future status-list__list-item status-list__list-item--future">
                 {% icon icon="circle_outlined" outlined=True %}
+                <span class="sr-only">{% trans 'Toekomstige status' %}</span>
```
Comment (pi-sigma) [inline src/open_inwoner/templates/pages/cases/statuses.html:72]:
```suggestion
                <span class="sr-only">{% trans "Toekomstige status" %}</span>
```

File: src/open_inwoner/templates/pages/cases/statuses.html [hunk 4]
```diff
@@ -77,6 +80,7 @@ <h3 class="utrecht-heading-3 utrecht-accordion__header">
         {% if case.end_statustype_data %}
             <li class="status--future status-list__list-item status-list__list-item--future">
                 {% icon icon="circle_outlined" outlined=True %}
+                <span class="sr-only">{% trans 'Toekomstige voltooide status' %}</span>
```
Comment (pi-sigma) [inline src/open_inwoner/templates/pages/cases/statuses.html:83]:
```suggestion
                <span class="sr-only">{% trans "Toekomstige voltooide status" %}</span>
```

**Review by pi-sigma — CHANGES_REQUESTED** [review]

File: src/open_inwoner/templates/pages/cases/statuses.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (jiromaykin) [reply src/open_inwoner/templates/pages/cases/statuses.html:16]: Oh well... Looking at the rest of the entire project there's quite some mixed use of quotes in these translation blocks, sometimes even within the same file (like in 'Plans>Preview') but I don't mind changing it here.

**Review by pi-sigma — APPROVED** [review]

**Pull request merged**